### ISSUE TITLE
Weird logo position

### ISSUE THREAD

**Issue by tertsdiepraam** [axodotdev/oranda#519]
Hi! Congrats on the release! I'm trying out this amazing project for uutils and it's great! 

I tried using the `logo` option, but the default position of the logo is a bit strange:

![image](https://github.com/axodotdev/oranda/assets/11643477/a324cc41-c4f3-4a6b-a476-e2de83046356)

The image in this example is an SVG, but the same placement happens for a PNG. The cause of the issue seems to be that elements with the `img` class are given `display: inline`, which doesn't play nice with the `margin: auto` on the logo.

As a workaround, I centered the image above the project name with the following additional CSS:
```css
.logo {
  display: block;
}
```

I would also like to suggest to give the logo some reasonable default size, instead of having it be determined by the image, so users don't have to change either the image or CSS to make the logo a reasonable size.

**Comment by ashleygwilliams** [comment]
hey @tertsdiepraam! sorry you ran into this and thanks for filing but glad you found a workaround. definitely agree that how we handle logos could be improved. i'm curious if you have thoughts on what the ideal handling of the logo would be? do you like having the logo centered above or would you prefer the ability to have it inline next to your project name?

**Comment by tertsdiepraam** [comment]
Thanks for the quick response!

I'm not sure what I would prefer actually. I've put the logo above the name right now, but that does take up a lot of space.

I've experimented a bit with different layouts. This is what I ended up with:

![image](https://github.com/axodotdev/oranda/assets/11643477/9c76f35a-ca97-4191-aabe-c59a2c1bdbaa)

Having it next to the name looks a bit off in my opinion, but a whole additional paragraph fits on the screen. I tried with two different sizes, the bigger logo has balances the large font of the project name better in my opinion:

![image](https://github.com/axodotdev/oranda/assets/11643477/18c3c1e2-8343-4639-8513-6599d21368e5)

![image](https://github.com/axodotdev/oranda/assets/11643477/0bded223-25cb-4916-8e04-42465bd9393b)

Another consideration is that the last two did not look good on mobile.
